config/helm/aws-node-termination-handler/values.yaml

```diff
@@ -259,22 +259,22 @@ daemonsetTolerations:
 linuxTolerations: []
 windowsTolerations: []
 
-# If the probes server is running for the Daemonset
+# If the probes server is running for the Daemonset. Only used in IMDS mode.
```

Comment:
To be more correct, let's remove "for the Daemonset", since the probes server can be used in a deployment and is not related to IMDS mode.

```suggestion
# If the probes server is running
```